app/layout/footer/component.tsx

```diff
@@ -31,9 +31,6 @@ export const Footer: React.FC<FooterProps> = () => {
             <Link href="/community">
               <a href="/community" className="text-gray-400 hover:text-gray-200">Community</a>
             </Link>
-            <Link href="/partners">
```

Comment:
I'm not 100% sure that we need to get rid of this page. I'll check it and let you know